packages/terra-time-input/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 * Fixed
   * Accessibility improvements.
+  * Supply isInvalid as well as aria-invalid to AccessibleInput
```

Comment:
This should be moved up to the Unreleased section.

Reply:
Fixed with 998f1f0f978b7efd32b26ea09259745a1c3b9583